### DISCUSSION TITLE
Getting Patient Representations from PatientDatabase and Event Timelines

### DISCUSSION THREAD

**Discussion by jordanhilsman** [som-shahlab/femr#189] [Q&A] [answered]
Hello, 

I am trying to get patient representations from the PatientDatabase class with event timelines generated by FEMR (The extract directory, previously in Tutorial 2A). I have generated the Patient event timelines, and have Trained the CLMBR model (Previously Tutorial 5). Do I need to have labels to get the representations from What used to be Tutorial 6? I do not need predictions from the models, but rather the representations generated for the patients by CLMBR.

Thank you!

**Comment by EthanSteinberg** [comment, marked as answer]
You need labels, but the value of those labels (true or false) can be arbitrary. It's recommended to just set the value to false. 

Note that the timing of those labels is important, as that defines when the representations are returned.

**Reply by jordanhilsman** [reply]
I'd like to capture as much of the Patient events in the representations as possible, should I capture the patient date of birth with
            'prediction_time': femr.pat_utils.get_patient_birthdate(patient)
            
 To do that? or should the prediction date be from the newest event?

**Reply by EthanSteinberg** [reply]
The prediction time should be the newest event to capture as much of the patient events in the representations as possible.
